clang/test/dpct/pytorch/user_defined_rule_pytorch.yaml

```diff
@@ -11,39 +12,118 @@
   In: ATen/cuda/CUDAContext.h
   Out: "<ATen/xpu/XPUContext.h>"
 
-- Rule: rule_c10_cuda_CUDAStream_h
+# c10::cuda macro rules
+- Rule: rule_c10_cuda_CUDAMacros_h
   Kind: Header
   Priority: Takeover
-  In: c10/cuda/CUDAStream.h
-  Out: "<c10/xpu/XPUStream.h>"
+  In: c10/cuda/CUDAMacros.h
+  Out: "<c10/xpu/XPUMacros.h>"
 
-- Rule: rule_c10_cuda_CUDAGuard_h
+- Rule: rule_c10_cuda_C10_CUDA_BUILD_SHARED_LIBS

```

Comment:
why need this Yaml file for the test?  or why not publish those rules?

Reply:
Yes, we can publish those rules instead